vitess.io/user-guide/sharding-kubernetes-workflow.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: doc
+title: "Sharding Workflow in Kubernetes"
```

Comment:
This title would not be consistent with the title of the other tutorial (`Sharding in Kubernetes`).

I suggest the following changes:
- let's use a title which is very similar to the navigation entry on the left
- let's replace the navigation entries from `(Codelab)` to `(Tutorial)` because we use "Tutorial" for the schema swap entry as well and it's the more common term in the outside world
- let's use the navigation entry names as title as well
- let's use the text ` (Tutorial, automated)` and ` (Tutorial, manual)` to distinguish between the two tutorial types
- let's reorder the entries such that the automated entry comes first in the navigaton

With that, I see the following entries in the `Sharding` navigation menu:
* Horizontal Sharding (Tutorial, automated)
* Horizontal Sharding (Tutorial, manual)
* Sharding in Kubernetes (Tutorial, automated)
* Sharding in Kubernetes (Tutorial, manual)

After these changes in the navigation, you could use `Sharding in Kubernetes (Tutorial, automated)` as title here.

Please update the titles in the other boiler plate .md files as well.

The other thing you need to do: At the beginning of each tutorial, you should briefly describe why it is the automated version and how it compares to the manual tutorial.